java/src/apps/AppsBase.java

```diff
@@ -238,12 +234,15 @@ protected void setAndLoadPreferenceFile() {
         }
         if (sharedConfig != null) {
             file = sharedConfig;
+            log.trace("Try preferences from sharedConfig {}", file.getPath());
```

Comment:
No need to change. But are these logging statements intended to be kept in this class?